pkg/elementalConfig/config.go

```diff
@@ -222,18 +222,21 @@ func NewInstallSpec(cfg v1.Config) *v1.InstallSpec {
 		recoveryImg.Source = v1.NewFileSrc(recoveryImgFile)
 		recoveryImg.FS = constants.SquashFs
 		recoveryImg.File = filepath.Join(constants.RecoveryDir, "cOS", constants.RecoverySquashFile)
+		recoveryImg.Size = constants.ImgSize
```

Comment:
This were defaults that were missing, hence the necessity of adding them in our shipped config.yaml for elemental under kairos.....

Reply:
yep yep, lots of duplicated code around, needs some refactoring